src/commons/support_bundle.py

```diff
@@ -57,7 +57,8 @@ def collect_upload_rotate_support_bundles(dir_path: str, max_sb: int = 0) -> tup
             return False, f"Failed to collect support bundles for {nodes}."
         cluster_obj = ClusterServices(host, user, password)
         status, response = cluster_obj.collect_support_bundles(sb_dir)
-        assert status, IOError("Failed to generate support bundles. Response: %s", response)
+        if not status:
+            raise AssertionError(f"Failed to generate support bundles. Response: {response}")
```

Comment:
All asserts are replaced with AssertionError. Any specific reason? asserts and raise AssertionError both do the same thing. Just that one extra line of code in raise.

Reply:
Basically assert is used in tests not in libs.